src/helper.py

```diff
@@ -40,6 +40,8 @@ def arguments_dictionary(lat, long, city, args):
         "forecast_days": get_forecast_days(args),
         "color": get_color(args),
         "gpt": 0,
+        "show_visibility": 1,
+        "show_cloud_cover": 1,
```

Comment:
We should set these to 0 as default, and show them if the user specifies in a cli argument (like `show_vis` or `sv`)